go/stellar/global.go

```diff
@@ -286,7 +286,7 @@ func (s *Stellar) handlePaymentNotification(ctx context.Context, obm gregor.OutO
 	if err = s.refreshPaymentFromNotification(ctx, msg.AccountID, msg.PaymentID); err != nil {
 		return err
 	}
-	s.G().NotifyRouter.HandleWalletPaymentStatusNotification(ctx, msg.AccountID, msg.PaymentID)
+	s.G().NotifyRouter.HandleWalletPaymentNotification(ctx, msg.AccountID, msg.PaymentID)
```

Comment:
what does this change do?

Reply:
`HandleWalletPaymentNotification` was never fired, i think this was incorrectly copy pasted during the change from ibms to oobms

Reply:
cc @patrickxb unless this was intentional?

Reply:
I can't remember if it was intentional.  @buoyad do you listen for both of these still?  payment_notification and payment_status_notification.

Reply:
No we don't listen to either of those anymore